packages/manager/apps/web/client/app/hosting/hosting.html

```diff
@@ -4,120 +4,158 @@
-    <div class="row">
-        <div class="col-sm-9">
-            <form name="formEditDisplayName" novalidate>
-                <div class="mb-3" data-ng-show="!edit.active">
-                    <div class="d-flex">
-                        <h1 class="d-inline-block mb-0 word-break">
-                            <span data-ng-bind="hosting.displayName"
-                                  data-ng-if="hosting.displayName"></span>
-                            <span data-ng-bind="::hosting.serviceDisplayName"
-                                  data-ng-if="!hosting.displayName"></span>
+<div class="oui-page-header" data-ng-if="!loadingHostingInformations && !loadingHostingError">
+    <div class="oui-page-header__container">
+        <div class="oui-page-header__content">
+            <div class="row">
+                <div class="col-sm-9">
+                    <form name="formEditDisplayName" novalidate>
+                        <div class="mb-3" data-ng-show="!edit.active">
+                            <div class="d-flex">
+                                <h1 class="d-inline-block mb-0 word-break">
+                                    <span data-ng-bind="hosting.displayName"
+                                        data-ng-if="hosting.displayName"></span>
+                                    <span data-ng-bind="::hosting.serviceDisplayName"
+                                        data-ng-if="!hosting.displayName"></span>
 
-                        </h1>
-                        <button class="btn btn-icon my-auto ml-2" type="button" title="{{ 'wizard_modify' | translate }}"
-                                data-ng-click="editDisplayName()">
-                            <span class="oui-icon oui-icon-pen_line oui-icon_small" aria-hidden="true"></span>
-                        </button>
-                    </div>
-                    <span class="font-italic" data-ng-bind="::hosting.serviceName"
-                           data-ng-if="hosting.serviceName !== hosting.displayName"></span>
+                                </h1>
+                                <button class="btn btn-icon my-auto ml-2" type="button" title="{{ 'wizard_modify' | translate }}"
+                                        data-ng-click="editDisplayName()">
+                                    <span class="oui-icon oui-icon-pen_line oui-icon_small" aria-hidden="true"></span>
+                                </button>
+                            </div>
+                            <span class="font-italic" data-ng-bind="::hosting.serviceName"
+                                data-ng-if="hosting.serviceName !== hosting.displayName"></span>
+                        </div>
+                        <div class="form-group my-4"
+                            data-ng-class="{'has-error': formEditDisplayName.txtDisplayName.$dirty && formEditDisplayName.$invalid}"
+                            data-ng-show="edit.active">
```

Comment:
I didn't touch this part, only the `oui-page-header` wrapper.
But why ? Because of the `form` ?

Reply:
Because `ng-show` will create a dom element and hide it. `ng-if` won't add the element to the DOM if I'm correct

Reply:
Yes I know, but my question was more oriented on why it's bad to have the DOM element here (because of the form ?) ;)
But I agree that most of the cases, a ngIf is sufficient.